Bottom: 2768f0fff6499f98dad289d13292c830d03afcda
Top:    9cc5d8faa6dffbecba016e3a93cb67ed60faa95d
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-07-18 15:32:19 +0800

Refresh of vhost_net-batch-update-used

---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index e163037..164ba4c 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -723,13 +723,10 @@ static void handle_tx(struct vhost_net *net)
 	vhost_disable_notify(&net->dev, vq);
 	vhost_net_disable_vq(net, vq);
 
-	if (vhost_sock_zcopy(sock)) {
-		printk("zcopy!\n");
+	if (vhost_sock_zcopy(sock))
 		handle_tx_zerocopy(net, sock);
-	} else {
-		printk("data copy!\n");
+	else
 		handle_tx_copy(net, sock);
-	}
 
 out:
 	mutex_unlock(&vq->mutex);
